crates/viewer/re_view_spatial/tests/tensor_1d.rs

```diff
@@ -0,0 +1,60 @@
+use re_chunk_store::RowId;
+use re_log_types::TimePoint;
+use re_types::{archetypes, datatypes};
+use re_view_spatial::SpatialView2D;
+use re_viewer_context::{ViewClass as _, ViewId, test_context::TestContext};
+use re_viewport::test_context_ext::TestContextExt as _;
+use re_viewport_blueprint::ViewBlueprint;
+
+fn x() -> Vec<f64> {
+    (0..100).map(|i| i as f64 * 100.0 / 99.0).collect()
+}
+
+#[test]
+fn test_tensor() -> anyhow::Result<()> {
+    let mut test_context = TestContext::new_with_view_class::<SpatialView2D>();
+
+    let tensor_data =
+        datatypes::TensorData::new(vec![1, 100], datatypes::TensorBuffer::F64(x().into()));
+    let image =
+        archetypes::Image::from_color_model_and_tensor(datatypes::ColorModel::L, tensor_data)?;
+    test_context.log_entity("tensor", |builder| {
+        builder.with_archetype(RowId::new(), TimePoint::STATIC, &image)
+    });
+
+    test_context.save_recording_to_file("tensor_1d_image.rrd")?;
```

Comment:
ignoring the added dependencies on test_context that's neat that you can do this now, but what is this used for?

those added dependencies do give me pause - I often run `cargo run nextest --all-features -p some_specific_crate` and all `some_specific_crate` that are view crates now have to build a bit more. It's probably not much, but worries me. Would be great to have this only where needed 🤔 (but I also really don't want to add even more feature flags :/)

Reply:
So my use case of adding it was:

1. Start writing tests
2. See black rectangle in .png
3. Wonder what would be shown in the viewer
4. `test_context.save_recording_to_file`
5. Open result in the viewer
6. Check if everything looks correct and potentially derive some meaningful values for `EyeControls3D` or VisualBounds.
7. Add these back into the tests.

But I can see your point too and given our compile times I'm happy to remove the method + deps again.

Reply:
Oh yeah, that line has to go